cmd/vulcan-exposed-ssh/main.go

```diff
@@ -189,12 +199,15 @@ func (r *runner) gradeVuln() ([]report.Vulnerability, error) {
 		if i.SSHLib == "libssh" {
 			matches := bannerRE.FindStringSubmatch(i.ServerBanner)
 			if len(matches) > 0 {
-				v, err := version.NewVersion(matches[1])
+				ver, err := version.NewVersion(matches[1])
 				// Don't stop parsing the results if a version hasn't been found.
 				if err == nil {
-					if c6.Check(v) || c7.Check(v) || c8.Check(v) {
-						libsshVuln.Details += fmt.Sprintf("* libssh version %v in port %v may be vulnerable\n", v.String(), i.Port)
-						vulnDb["libsshVuln"] = libsshVuln
+					if c6.Check(ver) || c7.Check(ver) || c8.Check(ver) {
+						v := libsshVuln
+						v.Details += fmt.Sprintf("* libssh version %v in port %v may be vulnerable\n", ver.String(), i.Port)
+						v.AffectedResource = fmt.Sprintf("%d/%s", i.Port, "ssh")
+						v.Fingerprint = helpers.ComputeFingerprint(v.Summary, v.Score)
```

Comment:
If the summary changes, the entire issue will change so it's not useful for the fingerprint. I'd rather use or empty args, or the `ServerBanner` value. WDYT?